index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="navbar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <ul>
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
+        </nav>
+    </header>
+        <section class="content-section">
+            <div class="content">
+                <h2 class="content-text">I'm Andrea Glad to see you!</h2>
+                <p class="content-text-p">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <p class="social-media">LET'S CONNECT</p>
```

Comment:
- [ ] This is a button that in the future will link to a contact page in your application. Using a `<a>` tag or `<button>` tag would be more appropriate and will help the users to redirect to that section of your project. Kindly change that.